server/services/licence/licenceStatus.ts

```diff
@@ -255,20 +257,24 @@ function getRiskManagementState(licence) {
   const checksConsideredAnswer = riskManagement?.hasConsideredChecks
   const awaitingInformationAnswer = riskManagement?.awaitingInformation || riskManagement?.awaitingOtherInformation
   const manageInTheCommunityAnswer = riskManagement?.manageInTheCommunity
-  const { proposedAddressSuitable, manageInTheCommunity } = riskManagement || {}
+  const pomConsultationAnswer = riskManagement?.pomConsultation
```

Comment:
boolean for use when determining if we should show the pom consultation warning.